views/jira-configuration-new.hbs

```diff
@@ -214,28 +214,31 @@
                           >
                             <span class="jiraConfiguration__appDropdownIcon">...</span>
                           </button>
-                          <aui-dropdown-menu class="" id="app-settings-dropdown-{{id}}">
-                            <aui-section>
-                              <!--TODO: need to add url for this-->
-                              <a
-                                  class=""
-                                  href="#"
-                                  data-installation-link="{{html_url}}"
-                                  target="_blank"
-                              >
-                                Edit
-                              </a>
-                              <!--TODO: need to add functionality for this-->
-                              <a
```

Comment:
Not sure why we were using links here because we're not navigating outside the app 🤷 Had to wrap each list item in a div as well otherwise clicking on one would fire the other (both at the same time).